modular_zubbers/code/modules/vending/_vending.dm

```diff
@@ -0,0 +1,56 @@
+/obj/machinery/vending
```

Comment:
Okay just gonna say we don't need this.
Limit the PR to just the clothing stuff and adding it to the vendors, we don't need this system when we can already do the same thing in a less clunky manner.